modules/tensor_mechanics/include/materials/RadialReturnStressUpdate.h

```diff
@@ -68,6 +69,10 @@ class RadialReturnStressUpdate : public StressUpdateBase
 
   MaterialProperty<Real> & _effective_inelastic_strain;
   MaterialProperty<Real> & _effective_inelastic_strain_old;
+  Real _max_inelastic_incr;
+
+private:
+  Real _max_inc;
```

Comment:
`protected` with `const` is always a 👍 . 
However many times it's not. As long as we are on the same page. It's always a fine line between making the framework accessible and protecting against corruption